src/character.cpp

```diff
@@ -1493,10 +1505,10 @@ int Character::get_working_arm_count() const
     }
 
     int limb_count = 0;
-    if( !is_limb_disabled( body_part_arm_l ) ) {
+    if( has_limb( body_part_arm_l ) && !is_limb_disabled( body_part_arm_l ) ) {
         limb_count++;
     }
-    if( !is_limb_disabled( body_part_arm_r ) ) {
+    if( has_limb( body_part_arm_r ) && !is_limb_disabled( body_part_arm_r ) ) {
         limb_count++;
     }
```

Comment:
i think the reason i did it this way was because i knew i was going to turn around and change it almost immediately in a follow up PR (the aforementioned manipulator score)